docs/src/description/constraints.md

```diff
@@ -34,56 +31,52 @@ public_inputs {
 
 boundary_constraints {
     # these are main constraints.
-    enf a.first = 0
-    enf a.last = 10
-
-    # these are auxiliary constraints, since they are defined against auxiliary trace columns.
-    enf p.first = 1
-    enf p.last = 1
+    enf a.first = 0;
+    enf a.last = 10;
 }
 
 integrity_constraints {
     <omitted for brevity>
 }
 ```
 
-### Public inputs and random values
+### Public inputs
 
-Boundary constraints can access public input values and random values provided by the verifier in their value expressions.
+Boundary constraints can access public input values provided by the verifier in their value expressions.
 
 To use public inputs, the public input must be declared in the `public_inputs` source section. They can be accessed using array indexing syntax, as described by the [accessor syntax rules](./syntax.md#section-specific-accessors).
 
-To use random values, they must be declared in the `random_values` source section. Random values are defined by an identifier which is bound to a fixed-length array (e.g. `rand: [16]`), but it is additionally possible to bind identifiers to specific random values or groups of values (e.g. `rand: [a, b[14], c]`). Random values can be accessed by using `$` followed by the array identifier and the index of the value (e.g. `$rand[10]`) or by using the name of the bound random value or group along with the index of the value within the group (e.g. `a` or `b[5]`), as described by the [accessor syntax rules](./syntax.md#section-specific-accessors).
-
-### Example of boundary constraints with public inputs and random values
+### Example of boundary constraints with public inputs and buses
```

Comment:
I would removed/reformulate ("... with ... and buses") as logically it is buses that are with or have boundary constraints